TBB.make

```diff
@@ -0,0 +1,28 @@
+
+ifndef COMPILER
+define compiler_help
+Set COMPILER to change flags (defaulting to GNU).
+Available compilers are:
+  GNU
```

Comment:
Can we add support for the Intel compiler too?

Reply:
Added INTEL/INTEL_LEGACY, another PR for adding this to all other models coming soon